src/page/product-added-modal.page.ts

```diff
@@ -4,7 +4,7 @@ export class ProductAddedModalPage {
   private checkoutButton: ElementFinder;
 
   constructor () {
-    this.checkoutButton = $('[style*="display: block;"] .button-container > a');
+    this.checkoutButton = $('a.btn.btn-default.button.button-medium');
```

Comment:
Try to find another selector, there is an attribute that makes this button unique